src/helpers/fileTypeIconMappings.json

```diff
@@ -0,0 +1,99 @@
+{
+  "7z": "package-x-generic",
```

Comment:
we should eventually load those from the original server, once we provide an endpoint or file to expose them